non_atomic_component_message_processing/section_subject_domain_of_dialog_system_messages/subject_domain_of_dialog_system_message_understanding/nrel_svo_triples.scs

```diff
@@ -0,0 +1,61 @@
+nrel_svo_triples
+<- sc_node_non_role_relation;
+<- relation;
+<- quasybinary_relation;
+<- oriented_relation;
+<- antireflexive_relation;
+<- asymmetric_relation;
+<- antitransitive_relation;
+=> nrel_main_idtf:
+    [тройки субъект-глагол-объект*]
+    (* <- lang_ru;; *);
+    [subject-verb-object triples*]
+    (* <- lang_en;; *);
+    [Subjekt-Verb-Objekt-Tripel*]
+    (* <- lang_de;; *);
+=> nrel_first_domain: concept_message;
+=> nrel_second_domain: concept_svo_triple;
+=> nrel_definitional_domain:
+    ...
+    (*
+    <= nrel_combination:
+        {
+        concept_message;
+        concept_svo_triple
```

Comment:
probably incorrect second domain
it should be a boolean of triple

Reply:
in the case of a quasi-binary relation, the second domain is a set whose elements belong to the tuples of the relation. in case of this relation, it connects message with a set (tuple) of triples, so i can't find the problem. can u exapin this a little?